src/scss/_mixins.scss

```
@@ -150,16 +150,23 @@
   width: $marker-dimension;
 }
 
-@mixin focusable($inset: false) {
+@mixin focusable($inset: false, $border-radius: .3em, $focus-element-box-shadow: 0 0 0 .1em rgba($color-focus, .4)) {
   &:focus {
     @if $inset == false {
       box-shadow: $focus-element-box-shadow;
     } @else {
       box-shadow: inset $focus-element-box-shadow;
     }
+
+    background-color: $color-focus;
+    backdrop-filter: blur(5px);
```

It seems adding the prefix property as well as a fallback increases the [support by ~4%](https://caniuse.com/css-backdrop-filter).

```suggestion
    -webkit-backdrop-filter: blur(5px);
    backdrop-filter: blur(5px);
```

---

Thx. Our build tooling does this for us automatically already.